### DISCUSSION TITLE
Change name of Turkey to new official name Türkiye

### DISCUSSION THREAD

**Discussion by lwohlhart** [laravel/nova-issues#5182] [Ideas & Feature Requests] [answered]
The countries name change to Türkiye was accepted and confirmed by the UN on 26 May 2022 https://www.un.org/en/about-us/member-states/turkiye

I first suggested the change in the Laravel-Lang/lang project where it's currently in review.

The suggestion there was to also create a pull request here because the projects synchronize the translation contents:

https://github.com/Laravel-Lang/lang/pull/2545#issuecomment-1364143383

Since I can't open a pull request here I'll provide the suggestion in this discussion - ( let me know if there's a better option for this )

```diff
@@ -338,7 +338,7 @@
   "Tonga": "Tonga",
   "Trinidad And Tobago": "Trinidad and Tobago",
   "Tunisia": "Tunisia",
-  "Turkey": "Turkey",
+  "Turkey": "Türkiye",
   "Turkmenistan": "Turkmenistan",
   "Turks And Caicos Islands": "Turks and Caicos Islands",
   "Tuvalu": "Tuvalu",
```

It'd be good if we wait to see if the pull request gets accepted there.

**Comment by crynobone** [comment, marked as answer]
will be available on upcoming release.
